notifications/http_test.go

```diff
@@ -62,15 +62,15 @@ func TestHTTPSink(t *testing.T) {
 		w.WriteHeader(status)
 	})
 	server := httptest.NewTLSServer(serverHandler)
-
 	metrics := newSafeMetrics()
 	sink := newHTTPSink(server.URL, 0, nil, nil,
 		&endpointMetricsHTTPStatusListener{safeMetrics: metrics})
 
+
 	// first make sure that the default transport gives x509 untrusted cert error
 	events := []Event{}
 	err := sink.Write(events...)
-	if !strings.Contains(err.Error(), "x509") {
+	if !strings.Contains(err.Error(), "x509") && !strings.Contains(err.Error(), "OpenSSL error")   {
```

Comment:
When using the unmodified version of go the error that is returned is 
"x509: certificate signed by unknown authority"
and while using the modified version the error that is returned is
"SSL routines:ssl3_read_bytes:tlsv1 alert unknown ca"